src/components/DopplerPlus/AgenciesForm.js

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { InjectAppServices } from '../../services/pure-di';
+import { Field } from 'formik';
+import {
+  EmailFieldItem,
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { useIntl } from 'react-intl';
+import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
+
+const AgenciesForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  // TODO: apply content
+  // TODO: add check to accept privacy policies
+
+  const volumeOptions = [
+    {
+      id: 'lessThan500k',
+      value: 'Menos de 500',
+      description: 'Menos de 500',
+    },
+    {
+      id: 'between500kAnd1m',
+      value: 'Entre 500k y 1m',
+      description: 'Entre 500k y 1m',
+    },
+    {
+      id: 'between1mAnd10m',
+      value: 'Entre 1m y 10m',
+      description: 'Entre 1m y 10m',
+    },
+    {
+      id: 'moreThan10m',
+      value: 'Más de 10m',
+      description: 'Más de 10m',
+    },
+  ];
+
+  const fieldNames = {
+    email: 'email',
+    firstname: 'firstname',
+    lastname: 'lastname',
+    phone: 'phone',
+    range_time: 'range_time',
+    volume: 'volume',
+  };
+
+  const getFormInitialValues = () => {
+    const initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
+
+    initialValues[fieldNames.email] = appSessionRef.current.userData.user.email;
+    initialValues[fieldNames.volume] = volumeOptions[0].value;
+
+    return initialValues;
+  };
+
+  const [formSubmitted, setFormSubmitted] = useState(false);
+
+  const validate = () => {
+    setFormSubmitted(false);
+  };
+
+  const onSubmit = async (values, { setSubmitting }) => {
+    setFormSubmitted(false);
+    try {
+      const result = await dopplerLegacyClient.requestAgenciesDemo(values);
+      if (result) {
+        setFormSubmitted(true);
+      }
+    } finally {
+      setSubmitting(false);
+    }
+  };
+
+  return (
+    <>
+      <Helmet>
+        <title>{'Agencies'}</title>
+        <meta name="description" content={'Agencies'} />
+      </Helmet>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <nav className="dp-breadcrumb">
+            <ul>
+              <li>
+                <a href={_('agencies.breadcrumb_url')}>{_('agencies.breadcrumb')}</a>
+              </li>
+              <li>{_('agencies.title')}</li>
+            </ul>
+          </nav>
+          <h2>Plan para agencias</h2>
```

Comment:
Even if you don{t have all correct contents, perhaps adding this into resource file is better for translation later. Or you can create a task for contents. Either way it is nice that each screen is as complete as it can be. (english/spanish)

Reply:
I have created a task for that. Due to I don't know what to put in this texts, I preferred to have something usefully in production to other people can see this.

Reply:
It is best that we try that all uploaded screens are as finished as it can be. So in english and spanish with some content, most of all because this is nearly finished. It's just an opinion though. 😄